src/main/java/ddingdong/ddingdongBE/domain/form/service/FacadeUserFormServiceImpl.java

```diff
@@ -0,0 +1,44 @@
+package ddingdong.ddingdongBE.domain.form.service;
+
+import ddingdong.ddingdongBE.domain.form.entity.Form;
+import ddingdong.ddingdongBE.domain.form.entity.FormAnswer;
+import ddingdong.ddingdongBE.domain.form.entity.FormField;
+import ddingdong.ddingdongBE.domain.form.entity.FormResponse;
+import ddingdong.ddingdongBE.domain.form.service.dto.command.CreateFormResponseCommand;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+@Transactional(readOnly = true)
+public class FacadeUserFormServiceImpl implements FacadeUserFormService {
+
+    private final FormResponseService formResponseService;
+    private final FormAnswerService formAnswerService;
+    private final FormService formService;
+    private final FormFieldService formFieldService;
+
+    @Transactional
+    @Override
+    public void createFormResponse(CreateFormResponseCommand createFormResponseCommand) {
+        Form form = formService.getById(createFormResponseCommand.form().getId());
+        FormResponse formResponse = createFormResponseCommand.toEntity(form);
+        FormResponse savedFormResponse = formResponseService.create(formResponse);
+
+        List<FormAnswer> formAnswers = toFormAnswers(savedFormResponse, createFormResponseCommand.formAnswerCommands());
+        formAnswerService.createAll(formAnswers);
+    }
+
+    private List<FormAnswer> toFormAnswers(FormResponse savedFormResponse, List<CreateFormResponseCommand.CreateFormAnswerCommand> createFormAnswerCommands) {
+        return createFormAnswerCommands.stream()
+                .map(formAnswerCommand -> {
+                    FormField formField = formFieldService.findById(formAnswerCommand.fieldId())
+                            .orElseThrow(() -> new IllegalArgumentException("해당 field를 id로 찾을 수 없습니다: " + formAnswerCommand.fieldId()));
```

Comment:
```suggestion
                    FormField formField = formFieldService.getById(formAnswerCommand.fieldId());
```

p3) 꼭 로직에 필요한 경우가 아니라면, 예외 처리는 해당 객체를 가져오는 서비스 내에서 처리하는 것이 좋습니다. FormFieldService내에서 예외처리를 해주세요.

Reply:
서비스 내에서 예외처리하도록 수정했습니다!